src/main/java/org/openrewrite/java/testing/assertj/AdoptAssertJDurationAssertions.java

```diff
@@ -0,0 +1,154 @@
+            J.MethodInvocation m = super.visitMethodInvocation(mi, ctx);
+            if (timeUnitMatchers.stream().anyMatch(matcher -> matcher.matches(mi))) {
+                return simplifyTimeUnits(mi, ctx);
+            }else if (ISEQUALTO_MATCHER.matches(mi)) {
+                return simplifyMultipleAssertions(mi, ctx);
+            }
+            return mi;
+        }
+
+        private J.MethodInvocation simplifyMultipleAssertions(J.MethodInvocation m, ExecutionContext ctx) {
+            if (!ISEQUALTO_MATCHER.matches(m)) {
+                return m;
+            }
+            Expression isEqualToArg = m.getArguments().get(0);
+            if (!ASSERT_THAT_MATCHER.matches(m.getSelect())) {
+                return m;
+            }
+
+            J.MethodInvocation assertThatArg = (J.MethodInvocation) ((J.MethodInvocation) m.getSelect()).getArguments().get(0);
+            if (!(assertThatArg instanceof J.MethodInvocation)) {
+                return m;
+            }
+            if (GET_NANO_MATCHER.matches(assertThatArg) || GET_SECONDS_MATCHER.matches(assertThatArg)) {
+                String methodName = assertThatArg.getSimpleName();
+                Expression assertThatArgSelect = assertThatArg.getSelect();
+                String formatted_template = String.format("assertThat(#{any()}).%s(#{any()});", methodMap.get(methodName));
+                return JavaTemplate.builder(formatted_template)
+                        .contextSensitive()
+                        .javaParser(JavaParser.fromJavaVersion().classpathFromResources(ctx, "junit-jupiter-api-5.9", "assertj-core-3.24"))
+                        .build()
+                        .apply(getCursor(), m.getCoordinates().replace(), assertThatArgSelect, isEqualToArg);
+            }
+
+            return m;
+        }
+
+        private J.MethodInvocation simplifyTimeUnits(J.MethodInvocation m, ExecutionContext ctx) {
+            int timeLength = 60;
+            String mName = m.getSimpleName();
+            if (mName.equals("hasMillis")) {
+                timeLength = 1000;
+            }else if (mName.equals("hasHours")) {
+                timeLength = 24;
+            }
+
+            Expression arg = m.getArguments().get(0);
+            Long argValue = SimplifyDurationCreationUnits.getConstantIntegralValue(arg); // note: guess my machine hasn't updated to the new commit yet, method should be public
+            if (argValue % timeLength == 0) {
+                // convert divided value to OpenRewrite LST type
+                int rawValue = (int)(argValue / timeLength);
+                J.Literal newArg = rawValueToExpression(rawValue);
+                // get new method name
+
+                // update method invocation with new name and arg
+                Expression methodSelect = m.getSelect();
+                return JavaTemplate.builder("#{any()}.#{any()}(#{int})")
```

Comment:
Just as a hint:
```suggestion
                return JavaTemplate.builder("#{any()}.#{any()}(#{any(int)})")
```

Reply:
ah whoops, I don't know why that slipped my mind. Thanks for catching it.